idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/tests/test_column.py

```diff
@@ -0,0 +1,297 @@
+
+            # Initialize column
+            m.fs.col.initialize()
+        elif Fpar == ProcessType.stripper:
+            m.fs.col.length_column.fix(12.1)
+            # Lean loading at the bottom of column @ final time
+            m.fs.col.loading = Param(initialize=0.17982818165156983)
+            for t in m.fs.time:
+                # Vapor
+                m.fs.col.vap_in_flow[t].fix(17.496)
+                m.fs.col.vap_in_temperature[t].fix(396.6)
+                m.fs.col.pressure_bottom[t].fix(183430)
+                m.fs.col.vap_in_mole_frac[t, "CO2"].fix(0.0145)
+                m.fs.col.vap_in_mole_frac[t, "H2O"].fix(0.9855)
+                # Liquid
+                m.fs.col.liq_in_flow[t].fix(84.48)
+                m.fs.col.liq_in_temperature[t].fix(382.15)
+                m.fs.col.liq_in_mole_frac[t, "CO2"].fix(0.0331)
+                m.fs.col.liq_in_mole_frac[t, "H2O"].fix(0.8547)
+                m.fs.col.liq_in_mole_frac[t, "MEA"].fix(0.1122)
+
+            # Initialize column
+            m.fs.col.initialize(
+                homotopy_steps_h=[0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1])
+        return m
+
+# ------------------------------------------------------------------------------
+    @pytest.mark.skipif(solver is None, reason="Solver not available")
+    @pytest.mark.integration
+    def test_steady_state_column_build(self, column_model_ss):
+
+        assert column_model_ss.fs.col.config.dynamic is False
+        assert column_model_ss.fs.col.config.liquid_side.transformation_scheme ==\
+            'FORWARD'
+        assert column_model_ss.fs.col.config.vapor_side.transformation_scheme == \
+            'BACKWARD'
+        assert degrees_of_freedom(column_model_ss) == 0
+
+
+    @pytest.mark.skipif(solver is None, reason="Solver not available")
+    @pytest.mark.integration
+    def test_dynamic_column_build(self, column_model_dyn):
+
+        assert column_model_dyn.fs.col.config.dynamic is True
+        assert column_model_dyn.fs.col.config.liquid_side.transformation_scheme ==\
+            'FORWARD'
+        assert column_model_dyn.fs.col.config.vapor_side.transformation_scheme == \
+            'BACKWARD'
+        assert degrees_of_freedom(column_model_dyn) == 0
+
+
+    @pytest.mark.skipif(solver is None, reason="Solver not available")
+    @pytest.mark.integration
+    def test_steady_state_column(self, column_model_ss):
+
+        m = column_model_ss
+        res = solver.solve(m)
+
+        # Solver status/condition
+        assert res.solver.termination_condition == TerminationCondition.optimal
+
+        # Outlet Stream Condition Testing
+        assert m.fs.col.vapor_phase.properties[0, 1].temperature.value ==\
+            pytest.approx(value(m.fs.col.vapor_exit_temp), abs=1e-1)
+
+        assert m.fs.col.liquid_phase.properties[0, 0].temperature.value ==\
+            pytest.approx(value(m.fs.col.liquid_exit_temp), abs=1e-1)
+
+        # CO2 mass balance
```

Comment:
I would like to see a few more assertions here. Notably, a set of excepted values for the outputs to make sure we keep seeing the same results, and some more comprehensive conservation tests (one for each species and one for energy).